e2e/vm/support_bundle_test.go

```diff
@@ -0,0 +1,37 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package vm
+
+import (
+	"os"
+	"strings"
+
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+	"github.com/runfinch/common-tests/command"
+	"github.com/runfinch/common-tests/option"
+)
+
+var testSupportBundle = func(o *option.Option) {
```

Comment:
Any other scenarios?

Reply:
I guess e2e tests are just the guard to prevent unexpected regression instead of something we believe it will never change?

Reply:
Yeah that's fair. I'll add the new scenarios.

Reply:
> And also what if one file is in both --include and --exclude?

We currently favor `--exclude` in this scenario, i.e. the file will not be included, as we want to ensure customers can exclude a file if they want to.

Reply:
Yeah I think it is good to cover it in documentation/CLI description and e2e tests.